docs/configuration.md

```diff
@@ -313,6 +315,65 @@ Given the config above, and assuming child config “d” is selected, the follo
 3. The result of 2 will be mixed into the base config
 4. The result of 3 will be the resolved config
 
+## Configuring loaders
+
+A loader is a script used by Intern to load suites into the testing environment. Only a single loader may be specified per
+ environment (Node or browser). Intern has a number of [loaders](https://github.com/theintern/intern/tree/master/src/loaders)
```

Comment:
This implies (to me) that Intern actually comes with various loaders. It might be better to say that Intern can _work with_ a number of loaders.

Reply:
I can see where that's a bit confusing for a loaderless architecture. What's the terminology we're using for the "loader" scripts?

Reply:
"Loader scripts" is what I've been using, but I'm open to suggestion.

Reply:
Loader scripts is good. I'm at a bit of a disadvantage in how to explain it because I'm familiar w/ the architecture. I think we'll be OK as long as we're consistent and take the time to highlight what a loader script does.